### PR TITLE
Upto sections

### PR DESCRIPTION
**Pull request by queden**
Added two sections on home page to show what I am currently doing and what I have done in the past.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
+    	<script src="script.js"></script>
+  	</head>
 
-  <body>
-    <div id="content">
+	<body>
+        <!-- Site nav bar-->
+        <nav>
+            <ul class="nav-list">
+                <li><strong>home</strong></li>
```
Comment (cshei) [inline portfolio/src/main/webapp/index.html:14]: nit: align with next line?  Unless this was intentional

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,22 +1,41 @@
...
+        <!--What I Was Up To-->
+        <section id="past-exp">
         </section>
-        <p>
-            I'm a rising sophomore at Vanderbilt University studying Computer Science, Math, and Engineering Management. This summer, I am interning (remotely) at Google 
-            on the <a href="https://cloud.google.com/edge-tpu">Edge TPU</a> team, as a part of their STEP program.
-        </p>
     </div>
-  </body>
+	</body>
```
Comment (cshei) [inline portfolio/src/main/webapp/index.html:56]: nit: revert this extra indentation

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (queden) [reply portfolio/src/main/webapp/index.html:14]: Good eye, not intentional. This should be fixed in most recent pull request

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (queden) [reply portfolio/src/main/webapp/index.html:56]: This also should be fixed in most recent pull request

**Pull request merged**